backend/experiment/models.py

```diff
@@ -27,7 +38,17 @@ def consent_upload_path(instance, filename):
 
 
 class Experiment(models.Model):
-    """A model to allow nesting multiple phases with blocks into a 'parent' experiment"""
+    """A model to allow nesting multiple phases with blocks into a 'parent' experiment
+    
+    Attributes:
+        slug (str): Slug
+        translated_content (list[ExperimentTranslatedContent]): Translated content
+        theme_config (theme.ThemeConfig): ThemeConfig instance
+        dashboard (bool): Show dashboard?
+        active (bool): Set experiment active
+        social_media_config (SocialMediaConfig): SocialMediaConfig instance
```

Comment:
Here the types *may* be necessary, but perhaps also worth checking if they aren't derived from Django's field class.

Reply:
Without this in the docstring the docs only contain the name of the class (and the complete source code behind the dropdown)